src/makeInfinityAdder.js

```diff
@@ -4,7 +4,21 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(num) {
+    if (typeof num === 'number') {
+      sum += num;
+
+      return adder;
+    }
+
+    const result = sum;
+
+    sum = 0;
+
+    return result;
+  };
 }
 
 module.exports = makeInfinityAdder;
```

Comment:
The module is being exported correctly, but ensure that it's being imported and called as expected in other files where it's used.